### DISCUSSION TITLE
Bandwidth Consumption

### DISCUSSION THREAD

**Discussion by mqshaw** [Xylopyrographer/STAC#38] [Q&A]
Hi Rob,

I hope that you have had a good few weeks, its been very busy for me here in Boise.  High school graduation and all that jazz.

Hey, have a question for you ... have you noticed an issue with the STAC's overloading the network at all?  We have been running 3 STACs now for the last 4 weeks, and for the last 2 we are getting packet loss in our stream up to YouTube.  You can "hear" it here :  https://youtu.be/Eyy2ni5bJKo?t=8569

Highly unlikely, but I was wondering if you have seen any issues on your end?  I am running the debug code, but I don't think that would have any impact on the network traffic, just the responsivity of the STAC.

I am going to try again next weekend ( this weekend is the 4th ), after rebooting the router and disconnecting the WiFi connection from the encoder to the router.

Mark

**Comment by Xylopyrographer** [comment]
Hi Mark. I was actually thinking about you a couple of days ago. Do pray you all are managing in this heat.

A bit of a frustrating situation for you. We've not seen an issue, but we have a network dedicated to just our streaming. It runs:
- an NDI feed from the FOH computer running ProPresenter 7.5
- 5 STAC's 
- the livestream (1080p) from our "studio" computer Restream.io which then pushes it to Facebook and YouTube.
- a remote connection from the studio computer to the FOH machine

All that is over Ethernet (except the STAC's of course).

Everything is fed into OBS and assembled there where it gets pushed to Restream.
Haven't metered the bitrate to see how close to the line we are.

The latest 1.9 version does respond a lot faster in the polling loop. Maybe slowing that down could help? Kind of comes back to adding an "Advanced Configuration" web page where things like that could be set dynamically. What do you think?

On other news, I've not had a chance to get back to the STAC project in a bit. We're doing some significant landscaping around the house and are trying to get a few things done before the new lawn shows up. Had to back burner that for this week due to the heat. But then with summer coming up and a bunch of our tech volunteers graduating and moving on and of course the usual summer vacation schedules to deal with, trying to fill all the slots. Ah well, it'll be what it'll be 😄.

Anyway, looking forward to getting back at it in a while (like fall).

Keep well!
